modules/ding_seo/ding_seo.info

```diff
@@ -0,0 +1,4 @@
+name = Ding SEO
+description = Provides various SEO functionality to ding2
+package = Ding!
+core = "7.x"
```

Comment:
The info file and install file is added in the main ding-seo branch, so just remove this before rebasing.